src/main/java/com/microsoft/azure/functions/HttpRequestMessage.java

```diff
@@ -22,10 +28,10 @@
     URI getUri();
 
     /**
-     * Returns the HTTP method name, such as "GET" and "POST".
-     * @return the HTTP method name, such as "GET" and "POST".
+     * Returns the HTTP method name as Enum
+     * @return type of HttpMethod 
      */
-    String getMethod();
+    HttpMethod getHttpMethod();
```

Comment:
This is going to be breaking change. Can you please provide notes in the PR with changes users need to make. Please see here as an example: https://github.com/Azure/app-service-announcements/issues/112